app/assets/v2/images/vision/triangle.svg

```diff
@@ -1 +1 @@
-2.455 1.666a.396.396 0 0 0-.165.252.383.383 0 0 0 .062.294 16.171 16.171 0 0 0 1.93 2.336.393.393 0 0 0 .557 0l2.098-2.095a.393.393 0 0 0 0-.556 12.408 12.408 0 0 1-1.48-1.793" fill="#180041"/><path d="M496.268 471.486c.332-.33.787-.534 1.204-.542.701.01 1.031.262 1.57.788l10.127 10.13c.378.386.917.61 1.482.613.513.027.99-.177 1.323-.51.333-.333.516-.805.512-1.324a2.088 2.088 0 0 0-.612-1.478l-16.27-16.272c-.642-.64-.722-1.093-.716-1.45.008-.43.216-.892.56-1.233.337-.338.78-.536 1.215-.544.385-.02.812.077 1.452.716l14.371 14.373c.378.386.917.61 1.481.613h.015c.514 0 .978-.18 1.31-.51.332-.334.514-.806.511-1.324a2.09 2.09 0 0 0-.612-1.478 13069.76 13069.76 0 0 0-12.897-12.898l-5.722-5.722c-.612-.61-.688-1.076-.68-1.45.007-.41.211-.849.56-1.198.35-.349.787-.553 1.2-.56.368.015.806.07 1.431.696l18.436 18.437c.377.387.916.611 1.48.614.531-.043.992-.178 1.325-.512.333-.333.514-.803.51-1.326a2.087 2.087 0 0 0-.61-1.474l-14.143-14.142c-.505-.44-.656-1.083-.767-1.505.008-.429.211-.884.542-1.215.338-.338.81-.553 1.234-.56.365.001.808.071 1.432.696l22.174 22.173c.377.388.917.612 1.48.617h.015c.515 0 .98-.182 1.31-.512.333-.333.514-.803.511-1.324a2.089 2.089 0 0 0-.61-1.478l-9.756-9.753c-.479-.41-.64-.988-.767-1.5.01-.433.218-.895.56-1.236.336-.336.78-.535 1.218-.542.406.015.835.067 1.43.661l10.167 10.168c4.012 4.01 6.003 8.45 5.917 13.188-.075 4.144-1.815 8.125-4.899 11.208-3.084 3.085-7.065 4.825-11.208 4.9-4.73.096-9.179-1.905-13.19-5.917l-16.47-16.469c-.608-.609-.686-1.039-.679-1.414.01-.432.211-.887.543-1.219zm32.52-17.146c-1.24-1.244-2.855-1.924-4.456-1.905a5.454 5.454 0 0 0-3.76 1.62 5.726 5.726 0 0 0-1.07 1.46l-8.279-8.28c-1.249-1.25-2.88-1.915-4.437-1.923a5.49 5.49 0 0 0-4.835 3.076l-.166-.168c-1.25-1.249-2.877-1.93-4.439-1.92-1.405.024-2.74.6-3.767 1.624-.936.981-1.59 2.495-1.63 3.768v.005c-.029 1.57.672 3.187 1.922 4.438l.168.17a5.645 5.645 0 0 0-1.44 1.052c-1.032 1.031-1.613 2.374-1.637 3.78-.027 1.572.681 3.196 1.94 4.454l1.96 1.96a5.772 5.772 0 0 0-1.462 1.07 5.436 5.436 0 0 0-1.619 3.76c-.028 1.572.673 3.19 1.922 4.44l14.496 14.494-6.236 6.234-28.437-28.439a17.935 17.935 0 0 1-5.288-12.766c0-9.956 8.1-18.054 18.054-18.054 4.823 0 9.357 1.877 12.767 5.287l2.63 2.63a.406.406 0 0 0 .555 0l2.628-2.63a17.94 17.94 0 0 1 12.767-5.287c9.955 0 18.055 8.098 18.055 18.054a18 18 0 0 1-.552 4.35l-6.354-6.354zm10.168 10.167l-.798-.797a21.55 21.55 0 0 0 1.288-7.366c0-12.025-9.782-21.807-21.807-21.807a21.659 21.659 0 0 0-15.42 6.387l-.253.253-.254-.253a21.655 21.655 0 0 0-15.42-6.387c-12.024 0-21.806 9.782-21.806 21.807a21.667 21.667 0 0 0 6.386 15.42l24.318 24.32c.011.007.02.017.031.026l6.466 6.466a.395.395 0 0 0 .556 0l8.635-8.639c4.49 4.2 9.813 6.513 15.136 6.513.112 0 .226 0 .337-.003 5.131-.092 10.027-2.213 13.79-5.973 3.748-3.75 5.863-8.642 5.956-13.77.105-5.736-2.433-11.488-7.14-16.197z" fill="#180041"/><path fill="#00ED92" d="M163.265 311.441l-5.583-9.875 10-22.375 17-.25-3.625 20.625zM485.765 279.191h22.5l6.834 20.833-7.334 13.167z"/></g></g></svg>
```

Comment:
Can you please cherry pick your changes and omit all of the changes to these `.svg` assets?

Reply:
Sure